portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -63,9 +65,11 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       long timestamp = (long) entity.getProperty("timestamp");
       String message = (String) entity.getProperty("message");
       FormSubmissions comment = new FormSubmissions(id, fname, lname, email, number, timestamp, message);
+
       comments.add(comment);
-      //commentCount -= 1;
+      commentCount -= 1;
     }
+
     response.setContentType("application/json");
     String commentsJSON = convertToJsonUsingGson(comments);
     response.getWriter().println(commentsJSON);
```

Comment:
In production code we don't include print statements (no need to remove) but we will enforce this in the capstone project